cmd/snap/cmd_set.go

```diff
@@ -0,0 +1,71 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/i18n"
+)
+
+type cmdSet struct {
+	Positional struct {
+		Snap         string   `positional-arg-name:"<snap name>" description:"the snap to configure (e.g. hello-world)"`
+		ConfigValues []string `positional-arg-name:"<config value>" description:"configuration value (key=value)" required:"1"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+func init() {
+	addCommand("set",
+		i18n.G("Set configuration for the given snap"),
+		i18n.G(`Set configuration for the given snap. This command accepts a
+			number of key=value pairs of configuration parameters`),
```

Comment:
Please have a look at the other cmd*.go files to see how we generally format the multiple helps.